components/automate-cs-ocid/package.meta

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "chef/automate-cs-ocid",
+  "uses_platform_scaffolding": true
```

Comment:
need clarity on this. can discuss.

Reply:
@YashviJain01 @daveaugustus Can you please explain what this boolean does - uses_platform_scaffolding?